db/db_test.cc

```diff
@@ -2871,12 +2871,13 @@ TEST_F(DBTest, DISABLED_FIFOCompactionWithTTLTest) {
         ASSERT_OK(Put(ToString(i * 20 + j), RandomString(&rnd, 980)));
       }
       Flush();
+      ASSERT_OK(dbfull()->TEST_WaitForCompact());
     }
-    ASSERT_OK(dbfull()->TEST_WaitForCompact());
     ASSERT_EQ(NumTableFilesAtLevel(0), 10);
 
-    // sleep for 5 seconds
-    env_->SleepForMicroseconds(5 * 1000 * 1000);
+    // sleep for 2 seconds
+    env_->SleepForMicroseconds(2 * 1000 * 1000);
```

Comment:
Can we use the simulated clock instead?

https://github.com/facebook/rocksdb/blob/master/db/db_test_util.h#L478-L480